ENGAGEHF/Education/Views/VideoPlayers/WebView.swift

```diff
@@ -0,0 +1,86 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+import WebKit
+
+
+// Note: This view prints the following errors to the console:
+//
+// Warning: -[BETextInput attributedMarkedText] is unimplemented
+// Failed to request allowed query parameters from WebPrivacy.
+//
+// However, this seems to be a bug on Apple's end and can be ignored:
+// https://stackoverflow.com/questions/78395514/failed-to-request-allowed-query-parameters-from-webprivacy
+//
+struct WebView: UIViewRepresentable {
```

Comment:
Nice, we should eventually move this into SpeziViews to reuse it across different applications. Would be a good future issue.